src/server/transaction.h

```diff
@@ -124,6 +126,10 @@ class Transaction {
   void LogJournalOnShard(EngineShard* shard, journal::Entry::Payload&& payload,
                          uint32_t shard_cnt) const;
 
+  // Rewrite parts of underlying command.
+  // Can be called only once per transaction.
+  void RewriteAutoJournal(const CommandRewritePairs& changes);
+
```

Comment:
I'm not sure yet: either rewrite or dispatch custom. Or dispatch rewritten.

I decided to "store" the rewrite because: 
- it was easier for the SET command that is doing its parsing inside the coordinator fiber
- I didn't want to introduce one more exception to autojournalling - it makes the hop callback more complicated